src/frontends/pytorch/src/op/full.cpp

```diff
@@ -186,7 +186,23 @@ OutputVector translate_empty(NodeContext& context) {
     return {empty};
 };
 
+OutputVector translate_new_empty(NodeContext& context) {
+    // aten::new_empty(Tensor self, SymInt[] size, *, ScalarType? dtype=None, Layout? layout=None, Device? device=None,
+    // bool? pin_memory=None) -> Tensor
+    num_inputs_check(context, 2, 6);
+    auto input = context.get_input(0);
+    auto sizes = context.get_input(1);
+    auto value = context.mark_node(v0::Constant::create(element::f32, Shape{}, {0}));
```

Comment:
Should it be a constant with a single value or an empty constant?

Reply:
It should be constant scalar with value 0, similarly like aten::empty.
Now when I'm looking into it, it should behave exactly like aten::new_zeros and aten::zeros, however, I think there is an easy to fix bug in current implementation of those op that fail tests, I'll try that and try to remove duplication if possible

Reply:
My bad, while aten::new_zero and aten::new_empty seem to have same scheme and can be covered by the same translation, there seem to be a difference in schema between aten::zero and aten::empty.